projects/ccd-case-ui-toolkit/src/lib/shared/services/fields/fields.utils.ts

```diff
@@ -351,6 +351,22 @@ export class FieldsUtils {
     return caseField.field_type.type === 'FlagLauncher';
   }
 
+  public static getValidationErrorMessageForFlagLauncherCaseField(caseField: CaseField): string {
+    if (caseField.display_context_parameter === '#ARGUMENT(CREATE)') {
```

Comment:
@johnbenjamin-hmcts - consider using a `switch()` statement to replace the multiple `if` conditions

Reply:
@Daniel-Lam - I have done the changes as you suggested.